### PR TITLE
RUN-5100 Fix the main function not called issue when using v2 api

### PR DESCRIPTION
**Pull request by licui3936**

### CODE REVIEW

File: src/api/window/window.ts [hunk 1]
```diff
@@ -506,6 +506,16 @@ export class _Window extends EmitterBase<WindowEvents> {
...
                 }
+                try {
+                    // this is to enforce a 5.0 contract that the child's main function
+                    // will not fire before the parent's success callback on creation.
+                    // if the child window is not accessible (CORS) this contract does
+                    // not hold.
+                    const webWindow: any = this.getWebWindow();
+                    webWindow.fin.__internal_.openerSuccessCBCalled();
+                } catch (e) {
+                    //console.error(e);
```
Comment (pbaize) [inline src/api/window/window.ts]: delete the commented out code

File: src/api/window/window.ts [hunk 2]
```diff
@@ -506,6 +506,16 @@ export class _Window extends EmitterBase<WindowEvents> {
...
                 }
+                try {
+                    // this is to enforce a 5.0 contract that the child's main function
+                    // will not fire before the parent's success callback on creation.
+                    // if the child window is not accessible (CORS) this contract does
+                    // not hold.
+                    const webWindow: any = this.getWebWindow();
+                    webWindow.fin.__internal_.openerSuccessCBCalled();
+                } catch (e) {
+                    console.error(e);
```
Comment (datamadic) [inline src/api/window/window.ts]: lets no log the error as this will fail in the expected cast most times

Comment (licui3936) [reply src/api/window/window.ts]: yeah, i tried to remove it. But the linting does't allow empty body. Can i just comment it out?

Comment (rdepena) [reply src/api/window/window.ts]: You can tslint ignore https://palantir.github.io/tslint/rules/no-empty/

Comment (licui3936) [reply src/api/window/window.ts]: I can change it. But i remember we use same tslint rules for core and js-adapter. Should we make them same?

**Pull request merged**